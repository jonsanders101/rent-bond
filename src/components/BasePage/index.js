import React from 'react';

export default ({ Headings, MainContent }) => (
  <div className="basepage">
    <div className="basepage__headings">
      <Headings />
    </div>
    <section className="basepage__main">
      <MainContent />
    </section>
  </div>
);
