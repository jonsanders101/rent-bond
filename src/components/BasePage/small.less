.basepage {
  height: 100%;
  display: flex;
  flex-direction: column;
}

.basepage__main {
  flex-grow: 2;
  overflow: scroll;
  margin: 2em;
  background: white;
  border-radius: 1em;
  padding: 1em;
}
