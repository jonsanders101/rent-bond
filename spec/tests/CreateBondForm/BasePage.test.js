import BasePage from '../../../src/components/BasePage';
import React from 'react';
import { shallow } from 'enzyme';

test('SHOULD correctly render <BasePage> component', () => {
  const component = shallow(
    <BasePage Headings="headings" MainContent="main" />
  );
  expect(component).toMatchSnapshot();
});
