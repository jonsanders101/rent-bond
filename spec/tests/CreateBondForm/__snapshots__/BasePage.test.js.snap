// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SHOULD correctly render <BasePage> component 1`] = `
<div
  className="basepage"
>
  <div
    className="basepage__headings"
  >
    <headings />
  </div>
  <section
    className="basepage__main"
  >
    <main />
  </section>
</div>
`;
